io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,54 @@
+id: io.neovim.nvim.tool.tmux
+build-extension: true
+sdk: org.freedesktop.Sdk//22.08
+runtime: io.neovim.nvim
+runtime-version: stable
+branch: stable
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: libevent
+    buildsystem: autotools 
+    config-opts:
+      - --prefix=${FLATPAK_DEST}
```

Comment:
```suggestion
```

you need a top level `build-options` to set the prefix.

Reply:
Fixed https://github.com/flathub/flathub/pull/4119/commits/dd1e4f1730ec18f1aaae415335ca1ea448d21d63